api-2.0/src/test/java/org/openmrs/module/fhir/api/RelatedPersonServiceTest.java

```diff
@@ -120,4 +128,19 @@ public void updateRelatedPerson_shouldCreateIfNotExists() throws Exception {
 		assertEquals(relationship.getPersonA(), updatedRelationship.getPersonA());
 		assertEquals(relationship.getPersonB(), updatedRelationship.getPersonB());
 	}
+
+	@Test
+	public void searchRelatedPersonByIdentifier_shouldReturnResourceIfExists() {
+		List<RelatedPerson> relatedPersonList = getService().searchRelatedPersonByIdentifier(PATIENT_IDENTIFIER);
+
+		assertNotNull(relatedPersonList);
```

Comment:
Might be good to verify more about the result than just if it's not null? Such as if it returned the correct related people, or has the correct relationship data?